protocol/go/go.mod

```diff
@@ -4,10 +4,11 @@ go 1.21
 
 require (
 	buf.build/gen/go/bufbuild/protovalidate/protocolbuffers/go v1.33.0-20240221180331-f05a6f4403ce.1
+	connectrpc.com/connect v1.17.0
```

Comment:
maybe add 

```
go.mod linguist-generated=false
```

to the .gitattributes file? Changes to go.mod files *should* be manually reviewed

Reply:
Didn't seem to work. I tried `protocol/go/go.mod` as well

Reply:
I am not going to solve this in the current pr.